src/intl/en/common.json

```diff
@@ -43,6 +43,7 @@
   "devcon": "Devcon",
   "developers": "Developers",
   "developers-home": "Developers' home",
+  "distributed-validator-technology": "Distributed Validator Technology",
```

Comment:
i think we might actually prefer to put this in the `staking` section - @wackerow?